NetKVM/Common/ParaNdis_Common.cpp

```diff
@@ -143,6 +147,10 @@ static const tConfigurationEntries defaultConfiguration =
     { "*UsoIPv4", 1, 0, 1},
     { "*UsoIPv6", 1, 0, 1},
 #endif
+#if MAX_FRAGMENTS_IN_ONE_NB
+    {"MaxFragmentsInOneNB",MAX_FRAGMENTS_IN_ONE_NB,16,256},
+    {"ExtraPagesInPerTXQ",17,17,256},
```

Comment:
The same magic number 17 again?
256 is probably should be define. AFAIU it is actually virtio_get_indirect_page_capacity() but we can't use it here, so let's make the define and validate at run time that it is not greater than virtio_get_indirect_page_capacity().
16 might be unreal so we can use the same define above for 17?
If wee them here then they should be configurable from outside and described in the INF. Otherwise we can just assign them.

Reply:
@ybendito ,Perhaps we should consider dropping the ExtraPagesInPerTXQ parameter? Because it seems to be quite difficult for users to understand. The reason I added it was because I saw in your previous comments that it would be preferable to have configurability, but I'm not sure if I misunderstood that. If we decide to do away with this parameter, what would be an appropriate value to set? My understanding is that we could just go with 17, since our current mechanism ensures successful retransmission when resources are insufficient, so we should focus on minimizing resource usage. I'm not sure if my understanding is correct.

Reply:
@ybendito  Alright, I understand now. By the way, what does "In" stand for? -> I used "In" incorrectly here; the meanings of "In" and "Per" are probably redundant